packages/react-native-gesture-handler/src/web/handlers/GestureHandler.ts

```diff
@@ -56,15 +59,36 @@ export default abstract class GestureHandler implements IGestureHandler {
   // Initializing handler
   //
 
-  protected init(viewRef: number, propsRef: React.RefObject<unknown>) {
+  protected init(
+    viewRef: number,
+    propsRef: React.RefObject<PropsRef>,
+    actionType: ActionType,
+    forAnimated: boolean
```

Comment:
Can you align this to match the behavior of Android and iOS after https://github.com/software-mansion/react-native-gesture-handler/pull/3646? `dispatchesAnimatedEvent` is now passed in the config of a gesture, so you should be able to update the relevant field in `updateGestureConfig`.

I think the detector shouldn't have any references to `dispatchesAnimatedEvent` besides reading it from gestures - https://github.com/software-mansion/react-native-gesture-handler/pull/3652.

Reply:
I think `forAnimated` isn't needed here anymore (and in other handlers where init is overriden)